### ISSUE TITLE
Feature request - Add the "Quick Fix" eclipse feature

### ISSUE THREAD

**Issue by ced75** [serge-rider/dbeaver#583]
Hi,

Is it possible to add the Eclipse "Quick Fix" feature in DBeaver?
It is very useful to quickly replace character strings in big scripts.

Thanks in advance.

**Comment by carlosjrcabello** [comment]
HI there,

I liked it! Another improvement could be `Alt + Shift + R` to rename all references of an table alias or highlighting them.

What do you think about it @serge-rider ?

**Comment by ced75** [comment]
Yes, and with the possibility to fix previous names, all names or following names depending on the carret position...
That's a very useful feature.

**Comment by serge-rider** [comment]
Agreed, that would be a great feature.
But here we go into refactoring zone. It is really complicated and requires a lot of effort. First of all existing back-end model of SQL editor doesn't allow such things. It has to be significantly improved.

I'll leave this feature request as a reminder. Can't say when I'll find time to implement it. 
I'm afraid I'll need to work on DBeaver on a full-time basis to support such things :)

**Comment by JessePreiner** [comment]
Would it be possible to give a rough outline of what would be involved? Maybe we can help out if we know what it would take!
